### ISSUE TITLE
working with Binary Option like IqOption

### ISSUE THREAD

**Issue by rrfaria** [Draichi/altcoin-bot-advisor#19]
First of all thanks for this project.

I'm learning RL and it can help a lot.

I'm begginer with gym and openai env. So my doubt is:

Is it possible to convert it to trade binary option of 1 minute ?
For instance:
Iqoption binary options allow you do performe some actions in 1, 5 or more minutes and if you choose right direction you earn percentage of this transation. In this case the payout is 85%

I already tried convetional machine learning like, svm, random forest, svc, xgboost but all doesn't performe good, then I tried deep learning with lstm and keras to classify those actions. worked but also not performing good.
maximum accuracy I got was 63% and running it directly on practice account I could not have and good return.
(there is a websocket where I connect to run directly on practice account)

I noticed one strategy doesn't performe for long time, so because of that I'm trying to learn o openai gym.

another doubt 

Could you pleaase teach me how gym works?
I fallowed same tutorial as you but I could not understand every thing

And I promise you I help you in your project if you teach me

![image](https://user-images.githubusercontent.com/11916455/90306801-13b8bc00-dea7-11ea-8f0e-93fd90f2222d.png)

**Comment by Draichi** [comment]
you need to check the cryptocompare api to get the data in the chosen granularity, or you can plug your own api.

right here
https://github.com/Draichi/T-1000/blob/337eb27b59852dc03eeecf7d3c6eb2929622e7b0/utils.py#L54

on the `get_datasets` function

 https://github.com/Draichi/T-1000/blob/337eb27b59852dc03eeecf7d3c6eb2929622e7b0/utils.py#L32
